game/board.py

```diff
@@ -308,90 +203,71 @@ def discover_cell(
-            char.transfer_tresaure(cell)
-            cell.remove_character()
-
-            return CORRECT_MOVE
-        else:
-            return self.make_move(dictionary)
+    # def move_to_own_character_position(self, current_player, row_to, col_to):
+    #     ch = self._board[row_to][col_to].character
+    #     if ch and ch.player == current_player:
+    #         raise moveToYourOwnCharPositionException()
+
+    # def is_a_player_character(self, row, col, current_player):
+    #     character = self._board[row][col].character
+    #     return character.player.name == current_player.name if character else False
+
+    # def is_valid_move(
+    #     self,
+    #     from_row,
+    #     from_col,
+    #     to_row,
+    #     to_col,
+    #     current_player: Player
+    # ):
+    #     coordinates = (to_row, to_col)
+    #     if not self.is_a_player_character(from_row, from_col, current_player):
+    #         raise notYourCharacterException()
+    #     if (coordinates not in posibles_positions(from_row, from_col)):
+    #         raise noPossibleMoveException()
+    #     self.move_to_own_character_position(current_player, to_row, to_col)
+    #     dictionary = {
+    #         "from_row": from_row,
+    #         "from_col": from_col,
+    #         "to_row": to_row,
+    #         "to_col": to_col,
+    #         "player": current_player,
+    #     }
+    #     return self.filter_move(dictionary)
+
+    # def make_move(self, dictionary):
+    #     from_row = dictionary["from_row"]
+    #     from_col = dictionary["from_col"]
+    #     to_row = dictionary["to_row"]
+    #     to_col = dictionary["to_col"]
+    #     current_player = dictionary["player"]
+    #     new_cell: Cell = self._board[to_row][to_col]
+    #     old_cel = self._board[from_row][from_col]
+    #     character: Character = self._board[from_row][from_col].character
+    #     character.player.arrows += new_cell.arrow
+    #     new_cell.transfer_tresaure(character)
+    #     new_cell.arrow = 0
+    #     new_cell.character = character
+    #     old_cel.character = None
+    #     self.discover_cell(to_row, to_col, current_player)
+    #     return CORRECT_MOVE
+
+    # def filter_move(self, dictionary):
+    #     cell_to = self._board[dictionary["to_row"]][dictionary["to_col"]]
+    #     current_player = dictionary["player"]
+    #     character_cel = cell_to.character
+
+    #     if cell_to.has_hole or self.has_opponent_player(character_cel, current_player):
+    #         row, col = dictionary["from_row"], dictionary["from_col"]
+    #         cell = self._board[row][col]
+    #         char = cell.character
+    #         self.discover_cell(dictionary["to_row"], dictionary["to_col"], current_player)
+    #         char.transfer_tresaure(cell)
+    #         cell.remove_character()
+
+    #         return CORRECT_MOVE
+    #     else:
+    #         return self.make_move(dictionary)
```

Comment:
Why we don't remove this lines?

Reply:
Yes, I forgot about these lines. I already remove it, thanks!